### PR TITLE
Clear command

### PR DESCRIPTION
**Pull request by jeannetoh99**
- Update Clear Command JavaDocs
- Add test cases for Clear Command
- Update Clear Command in UG and DG

### CODE REVIEW

**Review by underthehai — APPROVED** [review]
LGTM

**Review by lyeyixian — APPROVED** [review]
LGTM. Just the one in DG that haishan mentioned.

**Review by jiaweiteo — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -809,9 +809,9 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  User <ins>navigates to a specific travel plan (UC1)</ins>
-2.  User requests to view contacts
-3.  Wanderlust shows the list of contacts
+1.  User <ins>navigates to a specific travel plan (UC1)</ins>.
```
[the remark this answers is not in the document]
Comment (jeannetoh99) [reply docs/DeveloperGuide.md:812]: Yea we should! I was afraid someone else would have done it already so I didn't touch it first.

**Pull request merged**